client_tools.py

```diff
@@ -46,13 +45,17 @@ def parse_command_line():
 
     search_parser.add_argument('term')
 
-    parser.add_argument('-d', '--date', nargs="?", help='date to use', default=None)
+    parser.add_argument('-d', '--date', nargs="?", help='batch date in the form YYYYMMDD, the date which the batch was imaged.', default=None)
 
     parser.add_argument('-m', '--md5', nargs="?", type=str,  help='md5 batch to remove from database', default=None)
 
     parser.add_argument('-f', '--full_import', nargs="?", type=bool, help='Set to True if doing an '
                                                                            'import that imports both data and images',
                         default=False)
+    parser.add_argument('-e', '--existing_barcodes', nargs="?", type=bool, help="if True, skips creating skeleton"
```

Comment:
optional existing barcodes arg, which instructs the importer to skip importing image paths which do not already have collection collection object ids.